inc/classes/subscriber/Optimization/class-critical-css-subscriber.php

```diff
@@ -2,7 +2,7 @@
 namespace WP_Rocket\Subscriber\Optimization;
```

Comment:
```suggestion
namespace WP_Rocket\Engine\Optimization\CSS\CriticalPath;
```
We can move & rename the subscriber too